components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useState } from "react";
+import Modal from "../components/Modal";
+
+export interface WsStartEntry {
+    title: string
+    description: string
+    startUrl: string
+}
+
+interface StartWorkspaceModalProps {
+    visible: boolean;
+    recent: WsStartEntry[];
+    examples: WsStartEntry[];
+    selected?: Mode;
+    onClose: () => void;
+}
+
+type Mode = 'Recent' | 'Examples';
+
+function Tab(p: { name: Mode, selection: Mode, setSelection: (selection: Mode) => any }) {
+    const selectedTab = 'border-gray-800 text-gray-800';
+    const inactiveTab = 'border-none text-gray-400';
+    return <div onClick={() => p.setSelection(p.name)} className={"cursor-pointer py-2 px-4 border-b-4 " + (p.selection === p.name ? selectedTab : inactiveTab)}>{p.name}</div>
+}
+
+export function StartWorkspaceModal(p: StartWorkspaceModalProps) {
+    const [selection, setSelection] = useState(p.selected || 'Recent');
+
+    const list = (selection === 'Recent' ? p.recent : p.examples).map(e =>
+        <a key={e.title} href={e.startUrl} className="rounded-xl group hover:bg-gray-100 flex p-4 my-1">
+            <div className="w-full">
+                <p className="text-base text-gray-800 font-semibold">{e.title}</p>
+                <p>{e.description}</p>
```

Comment:
**issue**: This is showing some markdown prefix like `**JavaScript**`. Could we remove this part? It's fine to get this in as is for this milestone.